test/cpuinfo_x86_test.cc

```diff
@@ -36,12 +37,12 @@ class FakeCpu {
 
   uint32_t GetXCR0Eax() const { return xcr0_eax_; }
 
-  bool SysCtlByName(std::string name) const {
-    const auto itr = sysctlbyname_.find(name);
-    if (itr != sysctlbyname_.end()) {
-      return itr->second;
-    }
-    return 0;
+  bool GetDarwinSysCtlByName(std::string name) const {
+    return darwin_sysctlbyname_.count(name);
+  }
+
+  bool GetWindowsIsProcessorFeaturePresent(unsigned long ProcessorFeature) {
```

Comment:
why not using DWORD type here (to be consistent with the test line ~80) ?

Reply:
Good catch, this was before I included `windows.h` :)